docs: add SECURITY.md (#19)
charmed-hpc/filesystem-charms: SECURITY.md

# Security policy

## Supported versions

> Security updates will be released after vulnerabilities have been identified and fixed. At this point in time,
> we will only release security patches to the `latest/edge` and `latest/candidate` charm release channels.
> Additionally, any security patches for the `filesystem_info` charm library will only be released
> to the latest major version released on Charmhub.

Reviewer: IIRC I don't think we use the `latest/candidate` channel. Why don't we say `latest/stable` and `latest/edge` for now? We'll need to sit down and identify a long term support/maintenance strategy, but we could say that security fixes will be released for those channels currently:

```suggestion
Security updates for the filesystem charms will be released after vulnerabilities have been identified and fixed to the `latest/edge` and `latest/stable` release channels.

Security updates for the `filesystem_info` charm library will be released for all major versions that have had releases in the last year.
```

I tried to look at some other charm repositories to see what their support version policy is, but there doesn't seem to be an across-the-board standard.
